microbench/triangle-counting/CMakeLists.txt

```diff
@@ -86,7 +86,7 @@ add_subdirectory(src)
 
 # tests
 option(MICROBENCH_TESTING "Testing for microbenchmarks" ON)
```

Comment:
Enable testing for DRV-X

Reply:
what does this mean?

Reply:
Currently I think TC tests are missing the test_dummy and test_driver stuff that got DRVX tests working on pando-lib-gal?